### ISSUE TITLE
Keybindings customization

### ISSUE THREAD

**Issue by mcrampon** [hdlx/MacAppSwitcher#2]
Would be nice to be able to customize the keybindings. For instance, I'm used to `Super`+`Tab` instead of `Alt`+`~` to switch between windows of a given app.

Perhaps a small config file living next to the exe would do the trick? Automatically created with default values when running the exe for the first time?

Very nice project otherwise, I've been dying to find something like that for a while.

**Comment by hdlx** [comment]
Hey, thanks for the feedback. This would be nice indeed, I'll add this :-)

**Comment by hdlx** [comment]
Juste added this to the last release. Launching the exe should create a `MacAppSwitcherConfig.txt` file. The program must be relaunched to apply the new bindings.
Supported keys and format are briefly presented inside the created txt file.
It's a first draft, feedbacks are welcome :-)

**Comment by mcrampon** [comment]
Just tried it: works fine as far as I can see 👍 Thanks a bunch 🙇

**Comment by hdlx** [comment]
Hey ! Indeed, I overlooked the issue. I think I found a more robust solution. Key *release* only is bypassed now. This should let you use windows key as an "hold" key without interfering with start menu.

> Maybe the window swtich shouldn't exactly work the same way as the app swtich? In a sense, it doesn't really require holding because there is nothing to display. It's just a very normal combination of keys.

Indeed, not having any "hold" could work :-) Having a consistent behavior for hold key still is important as one might use window key for app switch, but from a UX perspective, it would be nice to have this option too.
